R/theme_gtsummary_roche.R

```diff
@@ -49,7 +49,7 @@ theme_gtsummary_roche <- function(set_theme = TRUE) {
   lst_theme <- lst_theme |>
     append(
       list(
-        "tbl_summary-fn:percent_fun" = gtsummary::label_style_number(digits = 1, scale = 100),
```

Comment:
Can you also update the description of the theme indicating this function is used for percent formatting?

Reply:
done!